iot-use-cases.md

```diff
@@ -29,7 +29,7 @@ Should you have any comments or questions please feel free to [Contact Us](/docs
                 <a class="img-button" href="/smart-energy/">
                     <div class="overlay">
                         <div class="eye">
-                            <img src="/images/eye-icon.svg">See more
+                            <img src="/images/eye-icon.svg" alt="See more">See more
```

Comment:
This is a wrong pattern of 'alt' tags. It should be something like "Smart energy use case details"